include/gmsh.h

```diff
@@ -1,16 +1,13 @@
 #ifndef MPM_POINT_GEN_GMSH_H_
 #define MPM_POINT_GEN_GMSH_H_
 
-#include <array>
 #include <fstream>
 #include <iostream>
 #include <limits>
 #include <map>
 #include <memory>
 #include <vector>
 
-#include <Eigen/Dense>
```

Comment:
Don't you need `#include <eigen3/Eigen/Dense>`

Reply:
I have added them back. Well, all the header files include `point.h` so I thought we don't have to include this anymore